### DISCUSSION TITLE
Minio to back up external AWS buckets?

### DISCUSSION THREAD

**Discussion by Rangel-Digital** [minio/minio#19509] [Q&A] [answered]
I currently has MiniIO installed on my TrueNAS box. I'm curious if it's possible to configure it to periodically back up Amazon or other AWS-based storage buckets. Can I use MinIO to periodically pull from other AWS objects?

**Comment by harshavardhana** [comment, marked as answer]
Use batch replication to replicate from remote source to local target

**Reply by Rangel-Digital** [reply]
I really need to pull from the local device to the S3 cloud storage... I don't want to push from the cloud to the local device.
